doc/classes/NavigationAgent2D.xml

```diff
@@ -171,6 +171,8 @@
 				- [code]type[/code]: Always [constant NavigationPathQueryResult2D.PATH_SEGMENT_TYPE_LINK].
 				- [code]rid[/code]: The [RID] of the link.
 				- [code]owner[/code]: The object which manages the link (usually [NavigationLink2D]).
+				- [code]link_start_position[/code]: If [code]owner[/code] is available and a [NavigationLink2D] will contain the global position to start moving through the link.
```

Comment:
```suggestion
				- [code]link_start_position[/code]: If [code]owner[/code] is available and the owner is a [NavigationLink2D], it will contain the global position of the link's point the agent is entering.
```